### DISCUSSION TITLE
可以在同一个版本下对模组进行不同分配

### DISCUSSION THREAD

**Discussion by shikongxingwen** [Meloong-Git/PCL#464] [提问] [answered]
对于内存溢出来的用户是一个很好的帮助
比如说我有版本相同，模组加载器相同，里面的模组不相同的A、B两个，把它两个结合在一起，选择方案A后它就切换为A版本的模组，选择方案B后就切换为B版本的模组，如何有相同的模组就合并

**Comment by Pigeon0v0** [comment, marked as answer]
可以交个新功能提案
